.jenkinsci/linux-post-step.groovy

```diff
@@ -2,18 +2,16 @@ def linuxPostStep() {
   timeout(time: 600, unit: "SECONDS") {
     try {
       // stop write core dumps
-      sh "bash -c 'ulimit -c 0'"
-      if (currentBuild.currentResult != "SUCCESS") {
+      sh "ulimit -c 0"
+      if (currentBuild.currentResult != "SUCCESS" && params.coredump) {
         // handling coredumps (if tests crashed)
-        def currentPath = sh(script: "pwd", returnStdout: true).trim()
-        def dumpsFileName = sprintf('coredumps-%1$s.tar',
+        def dumpsFileName = sprintf('coredumps-%1$s.bzip2',
           [GIT_COMMIT.substring(0,8)])
 
-        // sh(script: "find ${currentPath} -type f -name '*.coredump' | zip -j ${dumpsFileName} -@")
-        sh(script: "find ${currentPath} -type f -name '*.coredump' -exec tar -rvfj ${dumpsFileName} {} \\;")
+        sh(script: "find ${$WORKSPACE} -type f -name '*.coredump' -exec tar -rvfj ${dumpsFileName} {} \\;")
```

Comment:
Please verify this command actually works. I'm getting an error while verifying it locally:
```
$ find /tmp/test-tar/ -type f -name '123*' -exec tar -rvfj coredumps.bzip2 {} \;
tar: coredumps.bzip2: Cannot stat: No such file or directory
tar: Removing leading `/' from member names
/tmp/test-tar/1/2/1234.txt
tar: Exiting with failure status due to previous errors
tar: coredumps.bzip2: Cannot stat: No such file or directory
tar: Removing leading `/' from member names
/tmp/test-tar/1/2/123.txt
tar: Exiting with failure status due to previous errors
tar: coredumps.bzip2: Cannot stat: No such file or directory
tar: Removing leading `/' from member names
/tmp/test-tar/2/3/123.txt
tar: Exiting with failure status due to previous errors
```

Reply:
I would suggest `-exec tar cjvf ${dumpsFileName} {} \\+ \\;`, but please check the escape backslashes.